cypress/integration/user-control-panel.js

```diff
@@ -0,0 +1,42 @@
+if (Cypress.env('API') !== 'guillotina') {
+  describe('User Control Panel Test', () => {
+    beforeEach(() => {
+      // given a logged in editor
+      // and a folder that contains a document
+      // and the folder contents view
+      cy.visit('/');
+      cy.autologin();
+      cy.visit('/controlpanel/users');
+      cy.waitForResourceToLoad('@navigation');
+      cy.waitForResourceToLoad('@breadcrumbs');
+      cy.waitForResourceToLoad('@actions');
+      cy.waitForResourceToLoad('@types');
+    });
+
+    it.only('Should add User to controlPanel', () => {
+      // when I added a user from controlPanel
+      cy.get('.addSVG').click();
+      cy.get('input[id="field-username"]')
+        .clear()
+        .type('iFlameing');
+      cy.get('input[id="field-fullname"]')
+        .clear()
+        .type('Alok Kumar');
+      cy.get('input[id ="field-email"]')
+        .clear()
+        .type('ialokkumarsingh0@gmail.com');
```

Comment:
I would not recommend to use any real email address here and particularly not your own. :)

Just use "mail@example.com" or something similar.